crypto/dsa/dsa.go

```diff
@@ -13,6 +13,27 @@ const (
 	AlgorithmIDED25519   = eddsa.ED25519AlgorithmID
 )
 
+// BytesToPublicKey expects a non padded, base64 url encoded ssstring
+func BytesToPublicKey(algorithmID, base64Bytes string) *jwk.JWK {
+	if ecdsa.SupportsAlgorithmID(algorithmID) {
+		return &jwk.JWK{
+			KTY: "EC",
+			ALG: "ES256K",
+			CRV: "secp256k1",
+			X:   base64Bytes,
+			Y:   base64Bytes,
+		}
+	} else if eddsa.SupportsAlgorithmID(algorithmID) {
+		return &jwk.JWK{
+			KTY: "OKP",
+			ALG: "EdDSA",
+			CRV: "Ed25519",
+			X:   base64Bytes,
+		}
+	}
+	return nil
+}
```

Comment:
i'll toss up a PR shortly that handles conversion from bytes to public key. i'll push it down into the concrete crypto algorithm's implementation e.g. `secp256k1.go` and `ed25519.go`. Converting from public key bytes differs per algorithm and even for a given algorithm there's nuance. for example, `secp256k1` has both a compressed format and an uncompressed format which is indicated by the leading byte. 

if uncompressed, chop the leading byte, first 32 bytes are `x` and the 2nd 32 bytes are `y`. 
if compressed, the `x` value can be used to compute `y`

Reply:
PR up! https://github.com/TBD54566975/web5-go/pull/13